src/handler/global/personalstats.ts

```diff
@@ -0,0 +1,178 @@
+    const losses = formatter.format(
+        user.personalStats.assaults.lossesSustained
+    );
+    const destroyed = formatter.format(user.personalStats.facilities.destroyed);
+    const built = formatter.format(user.personalStats.facilities.spend);
+    return new MessageEmbed()
+        .setTitle(`${rank} ${user.name} [${user.id}]`)
+        .setDescription(
+            `${
+                formation
+                    ? getFormationStr(formation, user)
+                    : "Not currently in a Formation"
+            } | Motto: ${user.motto} | Elo: ${user.rating}`
+        )
+        .addField(
+            "Record",
+            `${
+                user.personalStats.wins.allies + user.personalStats.wins.axis
+            } wins \n ${user.personalStats.losses} losses`,
+            true
+        )
+        .addField(
+            "Damage",
+            `$${damage} damage done \n $${losses} losses taken`,
+            true
+        )
+        .addField("Points", `${getPoints(user)} total points`, true)
+        .addField("Networth", `$${await calculateNetworth(user, apiKey)}`, true)
+        .addField(
+            "Facilities",
+            `$${destroyed} worth destroyed \n $${built} worth built`,
+            true
+        )
+        .addField("\u200B", "\u200B", true)
+        .setColor(Color.BLUE);
+}
+
+function getPoints(user: UserData): number {
+    let points = user.points;
+    for (const skill in user.skills) {
+        if (skill === "queue") {
+            points -= 36;
+        }
+        const n = user.skills[skill as keyof Skills];
+        points += (n * (n + 1)) / 2;
+    }
+    return points;
+}
+
+function getFormationStr(formation: FormationData, user: UserData): string {
+    const position = getFormationPosition(formation, user.id.toString());
+    return `${position} of *${formation.name}* (${formation.members.length} members)`;
+}
+
+function getFormationPosition(formation: FormationData, id: string): string {
+    if (formation.leader === id) {
+        return "Leader";
+    } else if (formation.coleader === id) {
+        return "Co-Leader";
+    } else {
+        return "Member";
+    }
+}
+
+async function calculateNetworth(user: UserData, apiKey: string) {
+    try {
+        const units = await getUnits(apiKey);
+        const allUnits = await getAllUnits(apiKey);
+
+        let total = units
+            .map((unit) => {
+                const unitInfo = allUnits.find((u) => u.id === unit.id);
+                if (!unitInfo) {
+                    throw new Error("Unit not found");
+                }
+                return unitInfo.cost * unit.quantity;
+            })
+            .reduce((a, b) => a + b, 0);
+
+        total += user.funds;
+        total += user.reimbursement.amount;
+        total += checkForSoldUnits(user.reimbursement.fullInformation);
+        return numberWithCommas(total);
+    } catch (e) {
+        logger.error(e);
+    }
+}
+
+function numberWithCommas(x: number): string {
```

Comment:
why do this instead of using another `Intl.NumberFormatter`?

Reply:
copied straight from nw calculator without thinking tbh lol, fixed.